lib/elix/responders/gif_me.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Elix.Responders.GifMe do
+  @moduledoc """
+  Searches the Giphy public API and return the URL
+  of a GIF matching the search term.
+  """
+
+  use Hedwig.Responder
+
+  @base_url "http://api.giphy.com/v1/gifs/search"
+  @api_key "dc6zaTOxFJmzC" # Giphy’s public beta API key
+  @sample_size 10
+
+  @usage """
+  gif me <term> - Replies with a GIF URL matching the term
+  """
+  hear ~r/gif me (.+)/i, msg do
```

Comment:
Changing this to `(.*)`, it looks like what we're matching on is the full text of what the user typed, which means it doesn't currently matter whether or not you do "@elix gif me something" or just "gif me something" or "who wants some GIF meat?" I'm inclined to make the `@elix` bit optional, but I definitely want to add some word boundaries. Gotta watch out for that GIF meat...